packages/flex-plugin-scripts/src/utils/require.ts

```diff
@@ -38,14 +38,17 @@ export const hijack = (name: string, replacement: any) => {
 };
 
 /**
- * Runs craco
- * @param module  the module name
+ * Resolves the path to the given module. Appends the process.cwd node_modules as well
+ *
+ * @param module the module to resolve
  */
 /* istanbul ignore next */
-export const runCraco = (module: string) => require(module);
+export const resolve = (module: string): string => {
+  const paths: string[] = require.main && require.main.paths || [];
```

Comment:
y not get 😛

Reply:
I did try `get` and it didn't like it - we need to read more on `_.get` 😂